standard/statements.md

```diff
@@ -278,18 +278,20 @@ In addition to the reachability provided by normal flow of control, a labeled st
 
 ### 12.6.1 General
 
-A *declaration_statement* declares a local variable, local constant, or local function. Declaration statements are permitted in blocks, but are not permitted as embedded statements.
+A *declaration_statement* declares one or more local variables, a local constant, or a local function. Declaration statements are permitted in blocks, but are not permitted as embedded statements.
```

Comment:
```suggestion
A *declaration_statement* declares one or more local variables, one or more local constants, or a local function. Declaration statements are permitted in blocks, but are not permitted as embedded statements.
```

(I believe `const int x = 5, y = 10;` declares two local constants.)

Reply:
Fixed in upcoming commit